src/content/tools/pub/cmd/pub-unpack.md

```diff
@@ -0,0 +1,89 @@
+---
+title: dart pub unpack
+description: Downloads a package and unpacks its contents in place.
+---
+
+:::version-note
+The `unpack` subcommand was introduced in Dart 3.4.
+To download the archive of a package with an earlier SDK,
+visit the **Versions** tab of a package on the pub.dev site.
+:::
+
+_Unpack_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+```plaintext
+$ dart pub unpack <package>[:descriptor] [--[no-]resolve] [--output=<output directory>] [--[no-]force] [other options]
+```
+
+This command downloads the specified `<package>`
+according to the specified descriptor then
+extracts its contents to a `<package>-<version>` directory.
```

Comment:
Mentioning the `descriptor` first thing here made me think it's necessary to the command (like, the command functions _according_ to the descriptor makes it sound like it relies on that input), but reading a couple more sentences down shows that's not the case. I think leaving it out of this initial summary would be better:

```suggestion
This command downloads the specified `<package>` and
extracts its contents to a `<package>-<version>` directory.
```

Then, (if you remove the `--no-resolve` paragraphs like I suggested below) the very next thing will be the explanation of the `descriptor` anyway.